include/extractor/guidance/turn_instruction.hpp

```diff
@@ -43,8 +43,14 @@ enum TurnType // at the moment we can support 32 turn types, without increasing
     NewName,                // no turn, but name changes
     Continue,               // remain on a street
     Turn,                   // basic turn
+    FirstTurn,              // First of x turns
+    SecondTurn,             // Second of x turns
+    ThirdTurn,              // Third of x turns
     Merge,                  // merge onto a street
     Ramp,                   // special turn (highway ramp exits)
+    FirstRamp,              // numbered ramp
```

Comment:
Do you see any way of moving this additional information outside the turn type? I mean "only" handling four ramps at the same node is fine with me, but how will this evolve once we also emit intersection shapes?